doc/Extensions/Weathermap.md

```diff
@@ -32,3 +32,36 @@ Optional: If your install is in another directory than standard, set `$basehref`
 Automatically generate weathermaps from a LibreNMS database using WeatherMapper [Link](https://github.com/pblasquez/weathermapper)
 
 ![Example Network Weather Map](/img/network-weather-map.PNG)
+
+
+### Adding your Network Weathermps to the Dashboards
+
+Once you have created your Network Weather Map you can add it to a dashboard page by doing the following.
+
+### Step 1 
+When you create the Weathermap make sure to export as HTML and PNG you will need this for the out to the dashboard.
+
+In the Weathermap Plugin page, you will see the output. If you click on one of the maps it will show the image. In your URL bar, you see the entire URL, we will need to copy that.
+
+Example URL: `http://yourlibrenms.org/plugins/Weathermap/output/yourmap.html`
```

Comment:
The example url should be .png

Reply:
ah I see what you're doing here. Much better to just say to the user to right click the image and copy image url.